python/dialects/aie.py

```diff
@@ -290,7 +290,7 @@ def __init__(
         )
 
     def get_name(self):
-        return self.result.get_name()
+        return self.sym_name.value if self.sym_name else ""
```

Comment:
Good change. Before this, I had to have  `some_buf.get_name()[1:]` somewhere in my code, I think because it contained the `@` in front of the name. Good to be aware of this though in case it breaks other code anywhere (hopefully nobody did what I did).